settings.sample.py

```diff
@@ -1,3 +1,6 @@
+import logging.handlers
+import os
```

Comment:
Did you put this here intentionally or by auto-formatting?

While not PEP conform the reason I put these imports further down is so more unexperienced can get right to the settings without being overwhelmed.

Reply:
Sorry for answering late... 
The above changes were put by auto-formatting; I thought that it would be OK if all imports are at the top of the file...

Reply:
Ah dw; sorry for being a bit impatient from my side much rather.  

And you're right, it is okay, maybe I'm just too overly careful UX-wise…